src/status_im/transport/core.cljs

```diff
@@ -0,0 +1,66 @@
+(ns status-im.transport.core
+  (:require [cljs.spec.alpha :as spec]
+            [re-frame.core :as re-frame]
+            [status-im.constants :as constants]
+            [status-im.transport.message.core :as message]
+            [status-im.transport.filters :as filters]
+            [status-im.transport.utils :as transport.utils]
+            [status-im.utils.config :as config]
+            [taoensso.timbre :as log]
+            status-im.transport.inbox
+            [status-im.utils.handlers :as handlers]
+            [status-im.transport.db :as transport.db]))
+
+(defn initialize-offline-inbox [_]
+  (when config/offline-inbox-enabled?
+    (log/debug :init-offline-inbox)
+    {:dispatch [:initialize-offline-inbox]}))
+
+(defn init-whisper
+  [current-account-id {:keys [db web3] :as cofx}]
+  (log/debug :init-whisper)
+  (when-let [public-key (get-in db [:accounts/accounts current-account-id :public-key])]
+    (handlers/merge-fx cofx
+                       {:shh/add-discovery-filter {:web3           web3
+                                                   :private-key-id public-key
+                                                   :topic          (transport.utils/get-topic constants/contact-discovery)}
+                        :shh/restore-sym-keys {:web3       web3
+                                               :transport  (:transport/chats db)
+                                               :on-success (fn [chat-id sym-key sym-key-id]
```

Comment:
80line limit, same above

Reply:
not sure how we can respect the 80char per line limit while still having clean alignment for those kind of events

Reply:
putting it in variables would be one way